src/avio.jl

```diff
@@ -832,10 +832,10 @@ get_frame_presentation_time(stream, frame) = convert(Rational, stream.time_base)
 function get_frame_period_timebase(r::VideoReader)
     # This is probably not valid for many codecs, frame period in timebase
     # units
-    stream = get_stream(s)
+    stream = get_stream(r)
```

Comment:
This function was untested?

Reply:
Looks like it :sweat_smile:

Reply:
Yeah there's a few things like this in https://github.com/JuliaIO/VideoIO.jl/pull/409